src/app/@ansyn/ansyn/modules/menu-items/cases/models/case.model.ts

```diff
@@ -75,6 +75,7 @@ export interface IDilutedCaseState {
 	overlaysScannedAreaData?: IOverlaysScannedAreaData;
 	layers?: ICaseLayersState;
 	advancedSearchParameters?: IAdvancedSearchParameter;
+	runSecondSearch?: boolean;
```

Comment:
why is directly in the case state? why not inside of the advanced search params?